tests/zkevm/test_worst_memory.py

```diff
@@ -149,7 +197,13 @@ def test_worst_codecopy(
         pytest.param(1024 * 1024, id="1MiB"),
     ],
 )
-@pytest.mark.parametrize("fixed_dst", [True, False])
+@pytest.mark.parametrize(
+    "fixed_dst",
+    [
+        True,
+        False,
+    ],
+)
```

Comment:
Note that I did the `fixed_src_dst` change, but I later realized when double-checking runs that was incorrect. `RETURNDATACOPY` [**does not allow** being out of bounds](https://github.com/ethereum/go-ethereum/blob/v1.15.11/core/vm/instructions.go#L334-L336).

So basically I decided to keep the source at offset 0. I _could_ have made the "size" to compensate for the offset, but it felt to me this was against what the `size` parameter means since it wouldn't be true that we're doing a `RETURNDATACOPY` for `size` bytes.

Reply:
Oh wow, yes good point, I had not realized that the return buffer is treated differently :sweat_smile: